daras_ai_v2/gdrive_downloader.py

```
@@ -125,79 +126,95 @@ def wrapper(*args, **kwargs):
+        path = [file_id]
+        params = dict(alt="media", supportsAllDrives="true")
+
+    if connected_account_id is None:
+        http_resp = gdrive_file_api_get(path, params)
+        content = http_resp.content
+    else:
+        proxy_resp = composio_gdrive_file_api_get(connected_account_id, path, params)
+        if proxy_resp.data:
+            content = str(proxy_resp.data).encode()
+        else:
+            resp = requests.get(proxy_resp.binary_data.url)
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)